e2e/testcafe-devextreme/eslint.config.mjs

```diff
@@ -0,0 +1,256 @@
+        exceptions: ['#DEBUG', '#ENDDEBUG'],
+        markers: ['/'],
+      }],
+      '@stylistic/brace-style': ['error', '1tbs', {
+        allowSingleLine: true,
+      }],
+      'curly': ['error', 'multi-line', 'consistent'],
+      'unicode-bom': ['error', 'never'],
+      'eol-last': ['error', 'always'],
+      '@stylistic/indent': ['error', 4, {
+        SwitchCase: 1,
+        MemberExpression: 1,
+
+        CallExpression: {
+          arguments: 1,
+        },
+      }],
+      'quotes': ['error', 'single'],
+      'import/named': 2,
+      'import/default': 2,
+      'import/no-duplicates': 2,
+    },
+    plugins: {
+      '@stylistic': stylistic,
+      'import': importPlugin,
+    }
+  },
+  ...compat.extends('devextreme/typescript').map(config => {
+    const newConfig = {
+      ...config,
+      files: ['**/*.ts?(x)'],
+      ignores: ['**/*.d.ts'],
+    }
+
+    if (config.rules) {
+      newConfig.rules = changeRulesToStylistic(config.rules);
+    }
+
+    return newConfig;
+  }),
+  {
+    files: ['**/*.ts?(x)'],
+    ignores: ['**/*.d.ts'],
+    plugins: {
+      '@stylistic': stylistic,
+    },
+    languageOptions: {
+      parser: tsParser,
+      ecmaVersion: 6,
+      sourceType: 'module',
+      parserOptions: {
+        project: './tsconfig.json',
+        tsconfigRootDir: __dirname,
+      },
+    },
+    rules: {
+      'prefer-regex-literals': 'off',
+      'i18n/no-russian-character': ['error', {
+        includeIdentifier: true,
+      }],
+      '@typescript-eslint/no-unused-vars': ['error', {
+        "ignoreRestSiblings": true,
+        "caughtErrors": 'none',
+      }],
+      '@typescript-eslint/no-unsafe-function-type': 'off',
+      '@typescript-eslint/ban-types': 'off',
+      '@typescript-eslint/no-empty-object-type': 'off',
+      '@typescript-eslint/no-throw-literal': 'off',
+      '@typescript-eslint/switch-exhaustiveness-check': ['error', {
+        considerDefaultExhaustiveForUnions: true,
+      }],
+      'no-only-tests/no-only-tests': 'error',
+      'no-param-reassign': 'off',
+      'import/no-extraneous-dependencies': 'off',
+      'no-restricted-globals': 'off',
+      '@typescript-eslint/no-explicit-any': 'off',
+      '@typescript-eslint/explicit-function-return-type': 'off',
+      '@typescript-eslint/no-floating-promises': 'off',
+      '@typescript-eslint/no-unsafe-return': 'off',
+      '@typescript-eslint/no-non-null-assertion': 'off',
+      'no-await-in-loop': 'off',
+      '@stylistic/no-extra-parens': 'off',
+      '@typescript-eslint/require-await': 'off',
+      '@typescript-eslint/no-base-to-string': 'off'
+    },
+  },
+  ...compat.extends('devextreme/typescript').map(config => {
+    const newConfig = {
+      ...config,
+      files: ['**/*.d.ts'],
+    };
+
+    if (config.rules) {
+      newConfig.rules = changeRulesToStylistic(config.rules);
+    }
+
+    return newConfig;
+  }),
+  {
+    files: ['**/*.d.ts'],
+    plugins: {
+      '@stylistic': stylistic,
```

Comment:
Minor: '@stylistic/' rules aren't used in this block. Let's remove this plugin from here then

Reply:
deleted it